README.md

```diff
@@ -6,8 +6,53 @@ A Go HTTP client for the [MongoDB Atlas API](https://docs.atlas.mongodb.com/api/
 
 Note that `go-client-mongodb-atlas` only supports the two most recent major versions of Go.
 
+> NOTE: Atlas API v2 client is still under development. 
+> Please consider using v1 client: `go.mongodb.org/atlas/mongodbatlas` for production usage.
+
+
 ## Usage
 
+### Adding Dependency
+
+```
+go install go.mongodb.org/atlas
+```
+
+### Using in the code
+
+```go
+mongodbatlas import "go.mongodb.org/atlas/mongodbatlasv2/latest"
```

Comment:
why latest? are we expecting to keep every version on master?

Reply:
That reflects our current approach for the versioned API:

- mongodbatlasv2 (name of the client - new version of it)
- latest (metaname for the version of the versioned API we are using - currently 20230201)

If at some point we decide to host multiple versioned versions or name versions explicitly we would be able to do so.
Having just `go.mongodb.org/atlas/mongodbatlasv2` will hardcode generated client to that path and close our way to apply versioned api versions

Reply:
> If at some point we decide to host multiple versioned versions or name versions explicitly we would be able to do so.
> Having just go.mongodb.org/atlas/mongodbatlasv2 will hardcode generated client to that path and close our way to apply versioned api versions

what will block us from creating a new directory `latest` in the future if we will decide to have multiple versions? I don't understand why we cannot have only the `mongodbatlasv2` folder now and create a `latest` one In the future if needed. I agree with @fmenezes that `latest` is confusing for people who are not familiar with our versioning strategy.

Reply:
Ok. So basically this would be premature optimization. 
Changing to: 
go.mongodb.org/atlas/mongodbatlasv2

Reply:
Done in: [4b224e9](https://github.com/mongodb/go-client-mongodb-atlas/pull/410/commits/4b224e906fc4a9e000ca17ccfb4de5b2104eccaf)